src/ui/common/src/utils/shared.ts

```diff
@@ -12,11 +12,12 @@ export type LoadingStatus = {
   err: string;
 };
 
+export function isInitial(status: LoadingStatus) {
```

Comment:
Maybe it's just me, but these functions feel a little redundant to me.

Perhaps we can just remove them?

Reply:
hmmmm okay I somehow feel they look a bit cleaner..The caller only write `isSuccess(someComplexReduxState.status) || isFailure(someComplexReduxState.status)` which is slightly better look / readable than `someComplexReduxState.status.loading === LoadingStatusEnum.Success || someComplexReduxState.status.loading === LoadingStatusEnum.Failure`

Happy to clean them up in another PR if you feel the original way is a better practice :)